styles.css

```diff
@@ -485,6 +485,112 @@ nav {
   color: #fff;
 }
 
+.contact {
+  background-size: 100%;
+  position: absolute;
+  top: 670%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+}
+
```

Comment:
- Hey, please, let's improve on our design to match the Figma as much as we can to make your design consistent with the main design from the Figma:

*Your design*
![image](https://github.com/Obimbo07/Portfolio-mobileFirst/assets/59397159/b1295f3a-1a95-47ec-b57f-46b09676466c)

*Figma design*
![image](https://github.com/Obimbo07/Portfolio-mobileFirst/assets/59397159/364bb964-c4e3-4449-88a5-96ff1d7a92cb)